schedule/security/flows/default_aarch64.yaml

```diff
@@ -0,0 +1,57 @@
+---
```

Comment:
why did you need to create new default schedules? we don't plan to change those much, should be save to share among squad I think (if doesn't work, can be also duplicated later), did you find any keys that were different compare with what we have in maintenance? maybe it is the case as we don't do quarter updates but just to see the difference.

Reply:
I've cleaned it up and move to your defaults. Let's see how it will go with VRs :)

Reply:
the trick is to spread the complexity among the different solutions, you cannot get rid of it, so the yaml needs be a bit complex (with one single 1 default file, not many or having conditional schedule) the actions of testing can be done using the distribution pattern for example (it is a bit complex but not much) and the test data to validate can be a bit complex (having different files in yaml with test data depending on each arch for example) but not too complex (containing many if/elses).